sources/packages/api/src/route-controllers/student/student.students.controller.ts

```diff
@@ -187,7 +190,9 @@ export class StudentStudentsController extends BaseController {
   async synchronizeFromUserToken(
     @UserToken() studentUserToken: StudentUserToken,
   ): Promise<void> {
-    await this.studentService.synchronizeFromUserToken(studentUserToken);
+    if (studentUserToken.IDP === IdentityProviders.BCSC) {
```

Comment:
Did we decide that the synchronize must happen only for bcsc logins? Or is that part still TBD.?

Reply:
Yes, decided and persisted with the ticket.
"Review sync of user data upon login - Basic BCeID can NOT sync data"